Darwin has no SIGRTMIN (at least my Darwin doesn't).  I wouldn't call
this a real fix, it just makes SIG_SUSPEND undefined, which is happened
to be handled in the code that uses it.

--- include/private/gc_priv.h
+++ include/private/gc_priv.h
@@ -2117,11 +2117,13 @@
 #      define SIG_SUSPEND SIGPWR
 #    endif
 #   else  /* !GC_LINUX_THREADS */
+#    if !defined(GC_DARWIN_THREADS)
 #     if defined(_SIGRTMIN)
 #       define SIG_SUSPEND _SIGRTMIN + 6
 #     else
 #       define SIG_SUSPEND SIGRTMIN + 6
 #     endif
+#    endif
 #   endif
 #  endif /* !SIG_SUSPEND */
 
